src/ffi/ios.rs

```diff
@@ -20,6 +20,7 @@ pub unsafe extern "C" fn logger_init() {
         .with_ansi(false)
         .finish();
     tracing::subscriber::set_global_default(subscriber).expect("to be the only logger");
+    tracing::info!("Version 0.4.6");
```

Comment:
debug?

Reply:
yes, sorry. But I _would_ like to have something in there that references which version it's using. Would be really helpful for debugging...